<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitDetailsDock</class>
 <widget class="QDockWidget" name="UnitDetailsDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unit details</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="unit_name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="unit_profession">
      <property name="text">
       <string>Profession</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="unit_age">
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="inventory_view"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
